docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,14 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* Salespeople of companies
+* has a need to manage a significant number of contacts into either Leads or Contacts
```

Comment:
The double contacts seems a little odd. Maybe replace the first one with people, but it is less accurate.